src/DotLiquid/Context.cs

```diff
@@ -33,6 +33,29 @@ public class Context
         /// </summary>
         public SyntaxCompatibility SyntaxCompatibilityLevel { get; set; }
 
+        /// <summary>
+        /// Ruby Date Format flag, switches Date filter syntax between Ruby and CSharp formats.
+        /// </summary>
+        public bool UseRubyDateFormat { get; set; }
+
+        /// <summary>
+        /// Returns the CurrentCulture specified for this Context
+        /// </summary>
+        /// <exception cref="NotSupportedException">If the FormatProvider does not implement CultureInfo</exception>
+        public CultureInfo CurrentCulture
+        {
+            get
+            {
+                return this.FormatProvider is CultureInfo cultureInfo
+                    ? cultureInfo
+                    : throw new NotSupportedException("FormatProvider is not an instance of CultureInfo");

```

Comment:
Shouldn't the exception message string come from the resource file so it can be localized?

Reply:
No longer applicable